xds/src/main/java/io/grpc/xds/orca/OrcaOobUtil.java

```diff
@@ -66,27 +63,16 @@
 
 /**
  * Utility class that provides method for {@link LoadBalancer} to install listeners to receive
- * out-of-band backend cost metrics in the format of Open Request Cost Aggregation (ORCA).
+ * out-of-band backend metrics in the format of Open Request Cost Aggregation (ORCA).
  */
 @ExperimentalApi("https://github.com/grpc/grpc-java/issues/9129")
 public abstract class OrcaOobUtil {
```

Comment:
non-abstract and final. Add a private constructor to prevent external instantiation.